<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>494</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PyLogic</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QSplitter" name="splitter">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QGroupBox" name="groupBoxInput">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>1</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>Input</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QPlainTextEdit" name="textEditInput"/>
         </item>
        </layout>
       </widget>
       <widget class="QGroupBox" name="groupBoxOutput">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>4</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>Output</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="OutputTextEdit" name="textEditOutput">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton">
             <property name="text">
              <string>Clear</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tabPropositional">
        <attribute name="title">
         <string>Propositional</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="pushButtonCheckPropositional">
           <property name="text">
            <string>Check</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelNormalForms">
           <property name="text">
            <string>Normal Forms:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonNNF">
           <property name="text">
            <string>NNF</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonCNF">
           <property name="text">
            <string>CNF</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelProofProcedures">
           <property name="text">
            <string>Proof Procedures:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonResolution">
           <property name="text">
            <string>Resolution</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBoxExecutionTime">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>execution time</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tabFirstOrder">
        <attribute name="title">
         <string>First Order</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;Beta! Under construction&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ushButtonCheckFirstOrder">
           <property name="text">
            <string>Check</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>OutputTextEdit</class>
   <extends>QPlainTextEdit</extends>
   <header>pylogic.gui.widgets.h</header>
   <slots>
    <slot>workout()</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>actionAbout</sender>
   <signal>activated()</signal>
   <receiver>MainWindow</receiver>
   <slot>showAboutBox()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>339</x>
     <y>245</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>339</x>
     <y>245</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>textEditOutput</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>486</x>
     <y>446</y>
    </hint>
    <hint type="destinationlabel">
     <x>398</x>
     <y>408</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionOpen</sender>
   <signal>activated()</signal>
   <receiver>MainWindow</receiver>
   <slot>openFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>339</x>
     <y>245</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSave</sender>
   <signal>activated()</signal>
   <receiver>MainWindow</receiver>
   <slot>saveFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>339</x>
     <y>245</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonCheckPropositional</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>propositionalCheck()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>516</x>
     <y>110</y>
    </hint>
    <hint type="destinationlabel">
     <x>425</x>
     <y>466</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonNNF</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>propositionalNNF()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>516</x>
     <y>147</y>
    </hint>
    <hint type="destinationlabel">
     <x>561</x>
     <y>463</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonCNF</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>propositionalCNF()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>516</x>
     <y>184</y>
    </hint>
    <hint type="destinationlabel">
     <x>578</x>
     <y>461</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ushButtonCheckFirstOrder</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>firstOrderCheck()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>598</x>
     <y>137</y>
    </hint>
    <hint type="destinationlabel">
     <x>349</x>
     <y>246</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonResolution</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>propositionalResolution()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>561</x>
     <y>207</y>
    </hint>
    <hint type="destinationlabel">
     <x>540</x>
     <y>464</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>workout()</slot>
  <slot>showAboutBox()</slot>
  <slot>saveFile()</slot>
  <slot>openFile()</slot>
  <slot>propositionalCheck()</slot>
  <slot>propositionalNNF()</slot>
  <slot>propositionalCNF()</slot>
  <slot>firstOrderCheck()</slot>
  <slot>propositionalResolution()</slot>
 </slots>
</ui>
